src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    public static void doWork() {
```

Comment:
The method `doWork` should not be static. Static methods are not overridden in the same way as instance methods, which means polymorphism will not work as expected. Make this an instance method by removing the `static` keyword.